packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -103,13 +103,15 @@ const pollResult = async (runId: string) => {
 		.setPollAction(async () => pollAction(runId))
 		.setProgressAction((data: Simulation) => {
 			if (runId === props.node.state.inProgressOptimizeId && data.updates.length > 0) {
+				// TODO: Need to check if "first" is actually the latest update
 				const checkpointData = _.first(data.updates)?.data as CiemssOptimizeStatusUpdate;
```

Comment:
@Tom-Szendrey can you check that `first` here actually reflects the latest intermediate result? Recall that our data storage and fetch via hibernate is not reliable in terms of ordering, so typically we'd want to sort them manually .. somewhere...

Reply:
This was working as is but as you mentioned that the storage and fetch is not 100% reliable I have added the manual sort anyways.
I used `last` to keep consistent with our calibrate node's poller